common/text-level-semantics/index.json-ld

```diff
@@ -0,0 +1,35 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Text level semantics",
+		"fr": "Sémantique au niveau du texte"
+	},
+	"description": {
+		"en": "HTML elements used as is.",
+		"fr": "Les éléments HTML utilisé tel quel."
+	},
+	"modified": "2023-02-21",
+	"componentName": "text-level",
```

Comment:
This are not going to be relevant after the move inside the scaffolding folder, but for your info, the componentName need to match the folder name.